pytorch/pytorch-engine/src/main/java/ai/djl/pytorch/engine/PtEngine.java

```diff
@@ -53,6 +57,21 @@ static Engine newInstance() {
             }
             logger.info("Number of inter-op threads is " + JniUtils.getNumInteropThreads());
             logger.info("Number of intra-op threads is " + JniUtils.getNumThreads());
+
+            String paths = System.getenv("PYTORCH_EXTRA_LIBRARY_PATH");
```

Comment:
This has nothing to do with PyTorch, can user just simply do System.load() on the so files?

Reply:
1. PyTorch custom ops shared library usually depends on libtorch.so, and they installed on different path. This requires libtorch.so file is loaded first.

2. For DJL serving, there is no code can be executed before engine is loaded. A environment variable can allow user load custom ops easily.